### ISSUE TITLE
Thunderbird loses accounts on update

### ISSUE THREAD

**Issue by lx4r** [lukesampson/scoop-extras#3531]
I have installed Thunderbird via the `extras` bucket. Every time I update Thunderbird via Scoop all accounts I added are lost and I have to add them again. Also Thunderbird keeps asking whether the user wants to set it as the default email application upon every startup (that doesn't bother me, just thought it could be helpful 😇 ).
(Sorry for not adding more details but I just don't know which information could help to fix this.)

What can I do to persist my email accounts across updates of Thunderbird?
Thanks in advance for the help. :)

**Comment by mads256h** [comment]
Yeah i have the same issue. Thunderbird on scoop is worthless if it doesn't retain configuration from version to version.

**Comment by joegreene** [comment]
__DISCLAIMER__: I don't have thunderbird installed via `scoop`.

Not sure if this makes the difference but I noticed that `thunderbird-portable` has a `persist` property for the profile folder whereas `thunderbird` does not. Which one are you guys using?

Reference:
- [`thunderbird-portable`](https://github.com/lukesampson/scoop-extras/blob/master/bucket/thunderbird-portable.json)
- [`thunderbird`](https://github.com/lukesampson/scoop-extras/blob/master/bucket/thunderbird.json)

__Side note__: [This PR](https://github.com/lukesampson/scoop-extras/pull/2489) describes the history behind why both exist.

**Comment by mads256h** [comment]
I'm using the `thunderbird` one.

**Comment by lx4r** [comment]
I'm also using the `thunderbird` package.

**Comment by joegreene** [comment]
If you haven't yet you guys might want to try using `thuderbird-portable` instead. Hopefully that gets around the problem for now.

As for the actual problem at hand: I'm not really sure what causes it unfortunately. I remember having similar issues with `firefox` when I tried installing it via `scoop`. I know `thunderbird` utilizes the profiles folder located at `%APPDATA%\Thunderbird\Profiles` though.

__Some conjecture__:
Maybe each profile folder in `%APPDATA%\Thunderbird\Profiles` is in some way tied to each Thunderbird application install? If that's the case, and because `scoop` simply does a move/replace (instead of modifying a current install like the Thunderbird installer does), profiles could be becoming unrecognizable. Therefore, `scoop`'s `thunderbird` could be assuming none exist which means each new install will prompt you to import new profiles.

**Comment by lx4r** [comment]
@joegreene Okay, I'll try `thunderbird-portable`, thanks.  
I can confirm that Thunderbird uses `%APPDATA%\Thunderbird\Profiles` and (at least on my machine) creates a new profile after each update. In the newer versions it has even begun to show a popup telling the user that a new profile has been created.

Now I would like to test whether using `thunderbird-portable` solves the problem for now. Is there a way to force an update of an app in scoop?

**Comment by joegreene** [comment]
Thanks for the update @lx4r and I'm glad it's become a viable option. Does using `thunderbird-portable` work for you too, @mads256h?

In any case, I hope there's a way to fix `thunderbird` so it can retain profiles too. Perhaps link stuff from `%APPDATA%\Thunderbird\Profiles` to the `persist` folder? Seems like too much of a hack to get working though, so maybe not that. Otherwise, it seems like more of an annoyance to use than anything else.

**Comment by teohhanhui** [comment]
See https://github.com/lukesampson/scoop-extras/issues/4377#issuecomment-669390275 for the real cause.

**Comment by raycheung** [comment]
I'm using `thunderbird` and having this issue. My current workaround is right after the upgrade, start the app with `thunderbird -p` in command-prompt and select the previous profile.

**Comment by sdasda7777** [comment]
I'm using `extras/thunderbird` and this just happened to me yesterday. Could the installation script not just set the default profile as default automatically?
